js/get_random_values.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as msg from "gen/cli/msg_generated";
+import * as flatbuffers from "./flatbuffers";
+import * as dispatch from "./dispatch";
+
+function req(
+  typedArray: ArrayBufferView
+): [flatbuffers.Builder, msg.Any, flatbuffers.Offset, ArrayBufferView] {
+  const builder = flatbuffers.createBuilder();
+  const inner = msg.GetRandomValues.createGetRandomValues(builder);
+  return [builder, msg.Any.GetRandomValues, inner, typedArray];
+}
+
+/** Synchronously collects cryptographically secure random values. The
+ * underlying CSPRNG in use is Rust's `rand::rngs::ThreadRng`.
+ *
+ *       const arr = new Uint8Array(32);
+ *       crypto.getRandomValues(arr);
+ */
+export function getRandomValues<
+  T extends
+    | Int8Array
+    | Uint8Array
+    | Uint8ClampedArray
+    | Int16Array
+    | Uint16Array
+    | Int32Array
+    | Uint32Array
```

Comment:
In js/types.ts we have `TypedArray`

Reply:
But that `TypedArray` type is missing some integer-based typed array types and also encompasses `Float32Array` which browsers' `Crypto.getRandomValues()` don't allow.

Reply:
Can you add them?

Reply:
Ah ok